e2e/app/page/notebook-page.ts

```diff
@@ -237,6 +243,11 @@ export default class NotebookPage extends AuthenticatedPage {
     return frame.contentFrame();
   }
 
+  private async findRunButton(timeOut?: number): Promise<ElementHandle> {
```

Comment:
nit: I don't think O needs to be uppercased.

Also, I would add the time unit suffix so it's clear if this is milliseconds, seconds, nanoseconds, etc.

Reply:
parameter `timeout` is in a lot of functions. to keep it consistent, I'll make a separate PR to append the suffix.